src/api/wayfire/signal-definitions.hpp

```diff
@@ -320,6 +320,16 @@ using view_unmapped_signal = _view_signal;
  */
 using view_set_output_signal = _output_signal;
 
+/**
+ * name: keyboard-focus-changed
+ * on: view
```

Comment:
this signal doesn't seem to be emitted on the view, but on core.

Reply:
oh, I completely misunderstood what "on" means